test/installer/scripts/omsadmin_systest.rb

```diff
@@ -173,6 +174,13 @@ def test_unset_proc_limit_when_not_set
            "Process limit conf file was wrongly changed")
   end
 
+  def test_new_proc_limit
```

Comment:
nit: make this test name more descriptive: `test_set_proc_limit_with_end_label`

Reply:
Yes, it only happens when there is none for $AGENT_USER existing, so I think it needs to be more like:  test_set_new_proc_limit_anticipate_end_label.  The existing case does not care about ending labels.